src/components/Authentication/AccessControl/AccessControl.vue

```diff
@@ -9,8 +9,9 @@ export default {
   props: {
     roles: { type: Array, default: () => [] },
     role: { type: String, default: '' },
-    component_user_id: { type: Number, default: -2 },
-    user_id: { type: Number, default: -1 },
+    component_user_id: { type: Number, default: -1 },
```

Comment:
Can you explain the significance behind these default values?

Reply:
So the AccessControl component will render content is a user's ID matches up with a given component ID. I need some default values for those fields because if they're both null then they'll technically pass equality. So because I'm assuming we'd never use numbers below 0 for an ID, I chose -1 and -2 arbitrarily as placeholders